src/shared-content/administration/octopus-instance-mixed-os-warning.include.md

```diff
@@ -1,3 +1,7 @@
 :::div{.warning}
-Due to how Octopus stores the paths to various BLOB data (task logs, artifacts, packages, imports etc.), you cannot run a mix of both Windows Servers, and Octopus Linux containers connected to the same Octopus Deploy instance.  A single instance should only be hosted using one method.
+Due to how Octopus stores the paths to various BLOB data (task logs, artifacts, packages, imports, event exports etc.), you cannot run a mix of both Windows Servers, and Octopus Linux containers connected to the same Octopus Deploy instance.  A single instance should only be hosted using one method.
+:::
+
+:::div{.hint}
```

Comment:
I don't think this hint block is needed here

Reply:
Deleted